docker-compose.yml

```diff
@@ -0,0 +1,39 @@
+### Docker compose setup file
```

Comment:
Ideally, this file should be made for production, code autoreloading can be in an override file added by the user, and documented in `CONTRIBUTING.md`

Reply:
NOt necessarily, it can be used for development also, it gives much easier support when one only wants to develop with the bot, looking into the current situation, not after #32.

I also didn't make a contributing guide since there is none right now, once someone makes it, I will add the docker section to it.

Reply:
Line 3 and line 4.

A production docker file shouldn't have code autoreloading. That's my biggest concern here.